editors/turbo/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        magiblot turbo 8d42224c563865b232727e93169c81f11cd53fe1
+version             2024.01.07
+revision            0
+categories          editors sysutils
+license             MIT
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Experimental text editor based on Scintilla and Turbo Vision
+long_description    Turbo is an experimental text editor for the terminal, \
+                    based on the Scintilla code editing component by Neil Hodgson \
+                    and the Turbo Vision application framework.
+
+fetch.type          git
+
+if {${os.platform} eq "darwin" && ${os.major} < 14} {
+    # Lion+ (with Xcode 4.1+) have git; earlier need to bring their own.
+    # However Lion through Mavericks fail with SSL errors.
+    depends_build-append \
+                    port:git
+    git.cmd         ${prefix}/bin/git
+}
+
+post-fetch {
+    system -W ${worksrcpath} "git submodule update --init"
```

Comment:
```suggestion
    system -W ${worksrcpath} "${git.cmd} submodule update --init"
```

Reply:
here in this commit, I had to add the bit of code from fetch.tcl because the fetch.type was svn, so it wasn't automatically added. 

But if the fetch.type is git, as it is in this port, you should find that it works right without any additions.